contents/blog/posthog-vs-mixpanel.md

```diff
@@ -0,0 +1,925 @@
+---
+date: 2022-12-05
+title: PostHog vs Mixpanel
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+author: ["andy-vandervell"]
+categories: ["Guides", "Comparisons"]
+---
+
+Want to know how PostHog and Mixpanel are different? If you remember nothing else, remember these two points:
+
+1. Mixpanel is a product analytics tool.
+
+2. PostHog is an all-in-one platform that _includes_ product analytics **and** a whole bunch more.
+
+In this guide, we'll cover:
+
+- Mixpanel and PostHog's [core features](#core-features)
+- [Privacy and compliance](#privacy--compliance) on each platform
+- How they track usage and [library support](#tracking--sdks)
+- [Compare available integrations](#integrations) in multiple categories
+- [Answer frequently asked questions](#frequently-asked-questions) about PostHog and Mixpanel
+
+## How is PostHog different to Mixpanel?
+
+### 1. PostHog does more than product analytics
+
+Don't mistake us, product analytics is a core part of our platform, but it's just one cog in a suite of vital tools for product and data teams. When you adopt PostHog you also get session recording (with console logs), feature flagging, A/B and multivariate testing, and customizable data pipelines – all seamlessly integrated and with just one bill to pay. Don't want session recording? Just turn it off.
+
+### 2. It's built for engineers
+
+Anyone can use PostHog – just [ask our marketing team](/blog/posthog-vs-amplitude) – but it's designed for engineers first. We believe engineering-led companies are the future of tech and that future requires tools that let software engineers flex their product muscles. In building a platform [product engineers](/blog/what-is-a-product-engineer) love, we're also building a platform that gives data and marketing teams unrivaled insights.
+
+### 3. We're an open book
```

Comment:
I'd drop this as being not really relevant factor for many users (other than engineers, meaning you can bundle with the above). Instead, I'd add here either something more focused on the apps side (rather than us being a transparent company), or on the speed we ship at.

Reply:
Probably going to leave as-is for now, btw. I take your point, but combining 2 and 3 becomes a bit unwieldy.